opus/application/apps/results/views.py

```diff
@@ -612,6 +626,17 @@ def get_metadata(request, opus_id, fmt, api_name, return_db_names, internal):
             result_vals = result_vals[0]
             ordered_results = OrderedDict()
             for param_info in param_info_list:
+                if param_info.referred_slug is not None:
+                    referred_slug = param_info.referred_slug
+                    param_info = get_param_info_by_slug(referred_slug, 'col')
+                    param_info.label = param_info.body_qualified_label()
+                    param_info.label_results = (
+                                param_info.body_qualified_label_results(True))
+                    # Assign referred_slug, this will be used to determine if
+                    # the param info is from referred_slug, and we will used
+                    # the slug to get the metadata result later.
```

Comment:
Add period after "Assign referred_slug" and start new sentence. Also used -> use